web/src/components/ConsentBanner.test.js

```diff
@@ -39,4 +40,11 @@ describe('Consent banner component', () => {
       expect(onAgree.calledOnce).toEqual(true);
     });
   });
+
+  describe('accesibility', () => {
```

Comment:
@thetif Will these type of accessibility checks be added to all development tests in the future.

Reply:
yes that's the plan